server/db/seed.js

```diff
@@ -0,0 +1,179 @@
+import faker from 'faker';
+import campaignsService from '../db/services/campaigns';
+import contactListsService from '../db/services/contact_lists';
+import contactsService from '../db/services/contacts';
+import questionsService from '../db/services/questions';
+import scriptsService from '../db/services/scripts';
+import User from './models/users';
+
+function getRandomFourDigitInt() {
+  const floor = Math.ceil(1000);
+  const ceiling = Math.floor(10000);
+  return Math.floor(Math.random() * (ceiling - floor)) + floor;
+}
+
+const firstName1 = faker.name.firstName();
+const firstName2 = faker.name.firstName();
+const lastName1 = faker.name.lastName();
+const lastName2 = faker.name.lastName();
+const email1 = `${firstName1}.${lastName1}@notrealemail.com`;
+const email2 = `${firstName2}.${lastName2}@notrealemail.com`;
+const password = 'password';
+const userParams = [
```

Comment:
It would be nice to create 2 users (maybe root@beer.com and foo@bar.com with passwords as password. To help quickly test accounts. Of course, we can't directly feed in the password_hash so we may need to use the user service for this. What do you think?

Reply:
Don't need the user service. The way the user model is setup we can pass a string as password_hash and it gets hashed before it gets saved. I'll change the emails to be easy reference.